__tests__/data/iterable_source_output.json

```diff
@@ -0,0 +1,1066 @@
+[
+  {
+    "userId": "49431605de12d411bd21df8d909667b9",
+    "context": {
+      "integration": {
+        "name": "Iterable",
+        "version": "1.0.0"
+      },
+      "library": {
+        "name": "unknown",
+        "version": "unknown"
+      },
+      "traits": {
+        "email": "rohith@abcd.com"
+      }
+    },
+    "event": "emailSubscribe",
+    "integrations": {
+      "Iterable": false
+    },
+    "properties": {
+      "channelIds": [],
+      "createdAt": "2022-04-19 03:33:50 +00:00",
+      "emailListIds": [1589748],
+      "messageTypeIds": [],
+      "profileUpdatedAt": "2022-04-19 03:33:50 +00:00",
+      "publicIdString": "ad474bf7-e785-480f-b9d0-861b85ab5bf5",
+      "signupSource": "WebForm"
+    },
+    "receivedAt": "2022-04-19T03:33:50.000Z",
+    "timestamp": "2022-04-19T03:33:50.000Z",
+    "type": "track"
+  },
+  {
+    "message": "Unknwon event type from Iterable"
```

Comment:
Shouldn't these events pass?

Reply:
Yes, these events shouldn't pass. Payload-1, 2, and 3 are negative cases.